CHANGELOG.adoc

```diff
@@ -12,6 +12,17 @@
 | https://github.com/knative/client/pull/[#]
 ////
 
+## v0.15.1 (2020-06-03)
+
+[cols="1,10,3", options="header", width="100%"]
+|===
+| | Description | PR
+
+| 🐛
+| Update flag names to --request and --limit
+| https://github.com/knative/client/pull/872[#872]
+|===
+
```

Comment:
I think we can remove the v0.15.0 header as this version has never been officially released.

Reply:
yup, fixed.